lib/msal-angular/src/msal.broadcast.service.ts

```diff
@@ -3,24 +3,34 @@
  * Licensed under the MIT License.
  */
 
-import { Inject, Injectable } from "@angular/core";
-import { BehaviorSubject, Observable, Subject } from "rxjs";
-import { MSAL_INSTANCE } from "./constants";
+import { Inject, Injectable, Optional } from "@angular/core";
+import { BehaviorSubject, Observable, ReplaySubject, Subject } from "rxjs";
+import { MSAL_BROADCAST_CONFIG, MSAL_INSTANCE } from "./constants";
 import { EventMessage, EventMessageUtils, IPublicClientApplication, InteractionStatus } from "@azure/msal-browser";
 import { MsalService } from "./msal.service";
+import { MsalBroadcastConfiguration } from "./msal.broadcast.config";
 
 @Injectable()
 export class MsalBroadcastService {
-    private _msalSubject: Subject<EventMessage>;
+    private _msalSubject: Subject<EventMessage>|ReplaySubject<EventMessage>;
```

Comment:
`ReplaySubject` subclasses `Subject`, so I think you can leave this as-is.

Reply:
Got it. Changed.